ndcube/asdf/resources/schemas/ndcube-0.1.0.yaml

```diff
@@ -0,0 +1,25 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://sunpy.org/ndcube/schemas/NDCube-0.1.0"
+
+title:
+  Represents the ndcube NDCube object
+
+description:
+  Represents the ndcube NDCube object
+
+type: object
+properties:
+  data:
+    description: "Must be compatible with ASDF serialization/deserialization and supported by NDCube."
+  wcs:
+    tag: "tag:stsci.edu:gwcs/wcs-1.*"
+  extra_coords:
+    tag: "tag:sunpy.org:ndcube/extra_coords/extra_coords/extracoords-0.*"
+  global_coords:
+    tag: "tag:sunpy.org:ndcube/global_coords/globalcoords-0.*"
+
+required: [data , wcs]
+additionalProperties: true
```

Comment:
In ASDF how do the `allowAdditionalProperties` and `additionalProperties` differ? @braingram

Reply:
The asdf schemas are based off of "draft 4" jsonschema which doesn't define any behavior for `allowAdditionalProperties` (so defining this won't do anything). It does define `additionalProperties` (in a somewhat confusing way):
https://json-schema.org/draft-04/draft-fge-json-schema-validation-00#rfc.section.5.4.4

Reply:
Ah, thanks for clarifying, I guess [this](https://github.com/sunpy/sunpy/blob/8ac75969de24ee5b03db61560ff7ed4265c0d560/sunpy/io/special/asdf/resources/schemas/generic_map-1.2.0.yaml#L29C1-L29C33) needs to be updated then?

Reply:
Good catch, yeah a sunpy issue/PR to remove those is a good idea:
https://github.com/search?q=repo%3Asunpy%2Fsunpy%20allowAdditionalProperties&type=code

Reply:
Sure

Reply:
Opened PR #7686, which addresses the above issue.